docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,135 @@ sidebar_position: 5
+:::
+
+### GPS Rescue options
+
+- **Altitude mode**:
+  - **Maximum Altitude** - The craft will ascend to the maximum altitude during that flight plus `GPS_RESCUE_INITAL_CLIMB`.
+  - **Fixed Altitude** - The craft will ascend to a fixed altitude.
+  - **Current Altitude** - The craft will fly back at the altitude it was at when the failsafe occurred.
+- **Initial climb (meters)** - The distance the quad will climb, above the current altitude, when a rescue is initiated
+  and the altitude mode is set to **Current Altitude** also added when in **Maximum Altitude** mode.
+- **Return altitude (meters)** - The altitude the craft will fly back at when the altitude mode is set to **Fixed Altitude**.
+- **Ascent rate (meter/second)** - The rate at which the craft will ascend to the necessary altitude.
+- **Return ground speed (meters/second)** - The speed at which the craft will return to the home position.
+- **Maximum pitch angle** - The maximum pitch angle the craft will use to return to the home position.
+  Rescue throttle usually needs to be increased if the pitch angle is increased.
+  If pitch angle is too low, the craft might not be able to return in high winds.
+- **Descent rate (meters/second)** - The rate at which the craft will descend to the ground when it reaches the home position.
+- **Throttle minimum** - The **minimum** throttle value the craft will use to return to the home position.
+- **Throttle maximum** - The **maximum** throttle value the craft will use to return to the home position.
+- **Throttle hover** - The throttle value the craft will use to hover at the return altitude. **It is important to set this value accurately.**
+- **Minimum distance to home (meters)** - The minimum distance that the craft has to be from the home position to activate GPS Rescue.
+  If the rescue starts too close to home (within this minimum distance), the craft will fly away, on its current heading,
+  until at least this distance from home, and then start normal rescue behaviour
+- **Minimum satellites** - The minimum number of satellites required to activate GPS Rescue. If the number of satellites is below this value, GPS Rescue will not be activated.
+- **Allow arming without fix** - Allows arming without a GPS fix. If this is enabled and you have armed before the GPS fix is acquired,
+  the craft will disarm on loss of signal. If this is enabled, you'll not be able to take off without a GPS fix.
+- Sanity checks - These are used to prevent the craft from flying away or crashing.
+  - **On (highly recommended)** - The craft will perform sanity checks before starting the rescue. If the checks fail,
+    the craft will disarm instead.
+  - **Off** - The craft will not perform sanity checks before starting the rescue. This can lead to the craft flying away or crashing.
+  - **Failsafe only** - Sanity checks will only be performed on an actual loss of signal, will not be triggered if
+    you trigger GPS Rescue using the Aux mode.
```

Comment:
I suppose @haslinghuis meant to turn this into a table. Up to you, I think both a table and a list work fine for these

Reply:
IMO tables are easier to read.

Reply:
Okay, I can turn this into a table, but how would I handle nested cases such as `sanity checks` options?

Reply:
![image](https://github.com/user-attachments/assets/dc6f2dd0-2227-454a-b5d2-9b216340d737)

Perhaps this part could be separate on the bottom with 3 columns or use the nested options in first column.

Reply:
Or

| setting | description |
|:--- | :--- |
| sanity checks | these are used ...<br/>**on** text for on<br/>**off** text when off|
| failsafe only | sanity checks ... |

Reply:
```

| setting | description |
|:--- | :--- |
| sanity checks | These are used to prevent the craft from flying away or crashing.<br/><br/><ul><li>**on** The craft will perform saity checks before starting the rescue. If the checks fail, the craft will disarm instead.<br/></il><li>**off** The craft will not perform sanity checks before starting the rescue. This can lead to the craft flying away or crashing.</il></ul>|
| failsafe only | sanity checks ... |
```

Reply:
Formatted as a table in a90bde06d4eda7d54bf66760452b3420f32cf3eb

Reply:
Have converted configuration tab in the mean time